airflow/config_templates/config.yml

```
@@ -1633,6 +1633,20 @@ webserver:
       type: string
       example: ~
       default: "#51504f"
+    navbar_hover_color:
+      description: |
+        Define the color of navigation bar links when hovered
+      version_added: 2.8.1
```

This is a new feature; it should be added in the next minor version (2.9.0)
```suggestion
      version_added: 2.9.0
```